src/Prometheus/Storage/APC.php

```diff
@@ -170,6 +171,7 @@ private function collectCounters(): array
                 'help' => $metaData['help'],
                 'type' => $metaData['type'],
                 'labelNames' => $metaData['labelNames'],
+                'samples' => [],
```

Comment:
I'm not quite sure if this won't change the printed/restored result, can you check this?

Reply:
I'll do some tests, but the code is wonky and I'm not sure where it comes from and where its going, I'll dig a bit more.

Reply:
Got the blackbox testing up and running again, and it looks all good.

```
# HELP php_info Information about the PHP environment.
# TYPE php_info gauge
php_info{version="7.4.11"} 1
# HELP test_some_gauge it sets
# TYPE test_some_gauge gauge
test_some_gauge{type="blue"} 0
```

But I noticed 2 things I'll address:
- there are no tests for the renderer
- blackbox was outdated and not in the GHA workflow

Reply:
After further testing, I noticed that this behaviour of an empty `samples` array is also done in the `MetricsFamilySamples`, so doing it here will have no effect that won't already happen there.

Reply:
Okay thank you for digging more into it 👍